runtime/dart_vm_entry_points.txt

```diff
@@ -1,46 +1 @@
-dart:_internal,::,_printClosure
```

Comment:
You need to wait until your change in gen_snapshot is rolled into Flutter (or roll Dart before landing this PR).

Reply:
Yep.